network/overlay/delete_endpoint.go

```diff
@@ -34,7 +38,11 @@ func (m manager) DeleteEndpoint(ctx context.Context, n types.Network, e types.En
 
 	targetfd, err := netns.GetFromPath(e.TargetNetnsPath)
 	if err != nil {
-		return errors.Wrapf(err, "fail to get host namespace handler")
+		err := errors.Wrapf(err, "fail to get host namespace handle from path")
+		if !strings.Contains(err.Error(), "no such file or directory") {
+			return err
+		}
+		log.WithField("endpoint_netns_path", e.TargetNetnsPath).WithError(err).Error("ignore error")
```

Comment:
If we ignore it, is it an error, do we want a notification saying "ignore error" ?

Reply:
No you're right. Lowered to an "info" message